### PR TITLE
feat: add buyerFeedbackBatch data builder

### PR DESCRIPTION
**Pull request by GutierrezGo**
References https://autoricardo.atlassian.net/browse/CAR-6765

API ->  https://reporting-service.preprod.carforyou.ch/swagger-ui/index.html#/Reporting/getBuyerFeedbackBatchUsingGET

## Motivation and context

Extend the buyer feedback batch api-client

## Thank you 🐙

### CODE REVIEW

**Review by ufkaya — APPROVED** [review]
LGTM

File: src/types/models/buyerFeedbackBatch.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  id: number
+  name: string
+  location: {
+    city: string
+    zipCode: string
+  }
+}
+
+export interface BuyerFeedbackListing
+  extends Pick<ApiSearchListing, "id" | "make" | "model" | "price" | "type"> {
```
Comment (masone) [inline src/types/models/buyerFeedbackBatch.ts:13]: can you quickly explains what this TS magic does?

Comment (GutierrezGo) [reply src/types/models/buyerFeedbackBatch.ts:13]: With this utility `Pick<Type, Keys>` we pass an existing Type and set the keys that we need from the Type. Therefore, there's no need to repeat again the properties that we have already declared in another Type.

Comment (masone) [reply src/types/models/buyerFeedbackBatch.ts:13]: got it. so we end up exposing a Listing-ish but with a limited amount of fields

**Review by masone — APPROVED** [review]

**Pull request merged**